charts/kyverno-policies/templates/pod-sec-ctxt-capmc.yaml

```diff
@@ -0,0 +1,43 @@
+{{- $name := "pod-sec-ctxt-capmc" }}
+{{- if eq (include "kyverno-policy.podSecurityBaseline" (merge (dict "name" $name) .)) "true" }}
+{{- include "kyverno-policy.supportedKyvernoCheck" (dict "top" . "ver" ">= 1.6.0-0") }}
+apiVersion: kyverno.io/v1
+kind: Policy
+metadata:
+  name: {{ $name }}
+  namespace: services
+  annotations:
+    policies.kyverno.io/title: Mutate pod and container security contexts
+    policies.kyverno.io/category: Pod Security Standards (Baseline)
+    {{- if .Values.podSecuritySeverity }}
+    policies.kyverno.io/severity: {{ .Values.podSecuritySeverity }}
+    {{- end }}
+    kyverno.io/kyverno-version: 1.6.0
+    policies.kyverno.io/minversion: 1.6.0
+    kyverno.io/kubernetes-version: "1.22-1.23"
```

Comment:
CSM 1.2.0 will be running K8S 1.20. Not sure if there is a K8S update planned for 1.5 yet. 

So, why is this range here?

Reply:
I had copied it from the upstream repo. Let me remove it!